reanahub/reana-commons
# snakemake: preserve cwd when loading specifications

Related to reanahub/reana-server#462

How to test:
From outside the directory of one of the demo example (e.g. reana-demo-helloworld) run the following script:
```python
from reana_commons.specification import load_reana_spec
import os 

DEMO_DIR = "/path/to/reana-demo-helloworld"

print("cwd before load_reana_spec:", os.getcwd())
load_reana_spec(DEMO_DIR + "/reana-snakemake.yaml", DEMO_DIR)
print("cwd after load_reana_spec:",  os.getcwd())

```

Expected result: The CWD is preserved after loading the Snakemake specification
